A32NX/html_ui/Pages/VCockpit/Instruments/NavSystems/LogicElements/A32NX_FlightPlanManager.js

```diff
@@ -1053,6 +1054,91 @@ class FlightPlanManager {
+        let courseRad = course * (Math.PI / 180.0)
+        let parallelEntry = [courseRad, ((course + 110) % 360) * (Math.PI / 180.0)];
+        let tearEntry = [courseRad, ((course - 70) % 360) * (Math.PI / 180.0)]
+        let directEntry = [((course+110) % 360) * (Math.PI / 180.0), ((course + 290) % 360) * (Math.PI / 180.0)]
+
+        // let planeHeading = Simplane.getHeadingMagnetic();
+        let headingRad = planeHeading * (Math.PI / 180.0);
+        
+        let entry
+
+        if (headingRad > Math.min(parallelEntry[0], parallelEntry[1]) && headingRad < Math.max(parallelEntry[0], parallelEntry[1])) {
+            entry = "PARALLEL"
+        }
+        if (headingRad > Math.min(tearEntry[0], tearEntry[1]) && headingRad < Math.max(tearEntry[0], tearEntry[1])) {
+            entry = "TEAR"
+        }
+        if (headingRad > Math.min(directEntry[0], directEntry[1]) && headingRad < Math.max(directEntry[0], directEntry[1])) {
+            entry = "DIRECT"
+        }
+
+        let waypoints = [new LatLong(waypoint.infos.lat, waypoint.infos.long)] // start with the hold fix
+
+        let reverseTurnBearing = turn == "L" ? (course + 90) % 360 : (course - 90) % 360;
+        let turnBearing = turn == "R" ? (course + 45) % 360 : (course - 90) % 360;
+        let backCourse = (course + 180) %360
+
+        // switch(entry) {
+        //     case "PARALLEL": 
+        //         // fly backtrack for distance
+        //         let lastWP = Avionics.Utils.bearingDistanceToCoordinates(backCourse, distance, waypoint.infos.lat, waypoint.infos.long);
+        //         waypoints.push(lastWP)
+        //         // turn reverse to hold
+        //         lastWP = Avionics.Utils.bearingDistanceToCoordinates(reverseTurnBearing, 1, lastWP.lat, lastWP.long);
+        //         waypoints.push(lastWP)
+        //         // return to initial fix
+        //         lastWP = Avionics.Utils.bearingDistanceToCoordinates(course, distance, waypoints[0].lat, waypoints[0].long);
+        //         waypoints.push(lastWP)
+        //         break;
+        //     case "TEAR":
+        //         // fly 30* angle for distance from fix
+        //         let lastWP = Avionics.Utils.bearingDistanceToCoordinates((backCourse-30)%360, distance, waypoint.infos.lat, waypoint.infos.long);
+        //         waypoints.push(lastWP)
+        //         // continue normal turn to course
+        //         lastWP = Avionics.Utils.bearingDistanceToCoordinates(turnBearing, distance, lastWP.lat, lastWP.long);
+        //         waypoints.push(lastWP)
+        //         // return to fix on hold track
+        //         lastWP = Avionics.Utils.bearingDistanceToCoordinates(course, distance, waypoints[0].lat, waypoints[0].long);
+        //         break;
+        //     case "DIRECT":
+        //         // nothing to do here
+        //         break;
+        // }
+
+        // return waypoints;
```

Comment:
Can you clarify why this code is commented ?

If your PR is not ready yet, please make it draft 🙂

Reply:
Mentioned on the pr description. Let me get this commented out code onto another branch for now.